content/en/docs/concepts/workloads/controllers/ttlafterfinished.md

```diff
@@ -12,8 +12,8 @@ weight: 65
 
 The TTL controller provides a TTL (time to live) mechanism to limit the lifetime of resource
 objects that have finished execution. TTL controller only handles
-[Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) for
-now, and may be expanded to handle other resources that will finish execution,
+{{< glossary_tooltip text="Jobs" term_id="job" >}} for now,
+and may be expanded to handle other resources that will finish execution,
```

Comment:
nit: Can you reword or remove, `that will finish execution`?
nit: Is this possible (in TTL controller) in the near future? Is this claim relevant now?

Reply:
I might leave that for a separate PR.